packages/safe-ds-lang/src/language/generation/safe-ds-python-generator.ts

```diff
@@ -381,20 +395,238 @@ export class SafeDsPythonGenerator {
         )!;
     }
 
+    private getPlaceholderByName(block: SdsBlock, name: string): SdsPlaceholder | undefined {
+        return streamPlaceholders(block).find((placeholder) => placeholder.name === name);
+    }
```

Comment:
Could be useful elsewhere, so move to `nodeProperties.ts` (before/after `streamPlaceholders`).